DirectProgramming/C++SYCL_FPGA/Tutorials/DesignPatterns/onchip_memory_cache/README.md

```diff
@@ -1,29 +1,43 @@
-# Caching On-Chip Memory to Improve Loop Performance
+# `On-Chip Memory Cache` Sample
+
 This FPGA tutorial demonstrates how to build a simple cache (implemented in FPGA registers) to store recently-accessed memory locations so that the compiler can achieve II=1 on critical loops in task kernels.
 
+| Area                 | Description
+|:--                   |:--
+| What you will learn  | How and when to implement the on-chip memory cache optimization
+| Time to complete     | 30 minutes
+| Category             | Code Optimization
+
+## Purpose
+
+This sample demonstrates the following concepts:
+
+- How to implement the on-chip memory cache optimization technique
+- The scenarios in which this technique benefits performance
+- How to tune the cache depth
+
+## Prerequisites
+
 | Optimized for                     | Description
 ---                                 |---
-| OS                                | Linux* Ubuntu* 18.04/20.04 <br> RHEL*/CentOS* 8 <br> SUSE* 15 <br> Windows* 10
-| Hardware                          | Intel® Agilex™, Arria® 10, and Stratix® 10 FPGAs
+| OS                                | Ubuntu* 18.04/20.04 <br> RHEL*/CentOS* 8 <br> SUSE* 15 <br> Windows* 10
+| Hardware                          | Intel Agilex® FPGA <br> Intel® Arria® 10 FPGA <br> Intel® Stratix® 10 FPGA
```

Comment:
I actually prefer it the older way: Intel® Agilex™, Arria® 10, and Stratix® 10 FPGAs
Agilex, Arria 10 and Stratix 10 are FPGA families, so the intention is to say that the sample will work for any of these three family.
(Many different Agilex parts exist within the Agilex FPGA family, etc.)

Reply:
I changed it to match the formulations in the names database. In any event, I'll go with what you want.